.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
There's no simple way to fix it for `event_callback` and `url_verification` here: https://github.com/OperationCode/operationcode_bot/blob/b4e4418e3a1fc23c04d2104b0b46d3b0546d873d/operationcode_bot.rb#L50